assets/js/post_office_search.tsx

```diff
@@ -11,5 +17,17 @@ root.render(
     addressSearchURL={addressSearchUrl}
     locationsURL={locationsSearchUrl}
     onFoundLocations={() => {}}
+    resultsHeaderComponent={() => (
+      <Alert type="info" className="margin-bottom-4">
+        <strong>
+          {t('in_person_proofing.body.location.po_search.you_must_start.message', {
+            app_name: 'Login.gov',
+          })}
+        </strong>{' '}
+        <a href={t('in_person_proofing.body.location.po_search.you_must_start.link')}>
+          {t('in_person_proofing.body.location.po_search.you_must_start.link_text')}
+        </a>
+      </Alert>
+    )}
```

Comment:
@allthesignals Will we be using `InPersonLocationFullAddressEntryPostOfficeSearchStep` as the entry point rather than `AdressSearch` in Help Center?